openquake/commonlib/tests/logictree_test.py

```diff
@@ -476,6 +476,38 @@ def test_b_gr_relative_wrong_format(self):
         self.assertEqual(exc.message, 'expected single float value',
                          "wrong exception message: %s" % exc.message)
```

Comment:
Please disregard my comment, Graeme. I see now that we have a custom class LogicTreeError with a `.message` attribute, so it was correct as it was. This is not the deprecated attribute `.message` inherited from `Exception`.